lib/w1_Tuesday_Iteration.rb

```diff
@@ -0,0 +1,74 @@
+def benchmark(value = 1)
+  start_time = Time.now
+  value.times {yield}
+  end_time = Time.now
+  (end_time - start_time)
+end
+
+def race(arr, val = 1)
+  winner = nil
+	min_runtime = 1.0 / 0 #Infiinty
+	runtime = 0
+
+	arr.each_with_index do |procedure, index|
+		runtime = benchmark(value)
+
+		if runtime < min_runtime
+			min_runtime = runtime
+			winner = index
+		end
+	end
+
+	arr[winner]
```

Comment:
it looks like you might be missing an `end` here

Reply:
actually this entire method seems to be indented improperly

Reply:
Yeah missed this one
Will put a fix to a next PR